addons/l10n_fr/data/account_chart_template_data.xml

```diff
@@ -21,7 +21,9 @@
       <field name="expense_currency_exchange_account_id" ref="pcg_666"/>
       <field name="property_tax_payable_account_id" ref="pcg_44551"/>
       <field name="property_tax_receivable_account_id" ref="pcg_44567"/>
-      <field name="default_pos_receivable_account_id" ref="fr_pcg_recv_pos" />
+      <field name="default_pos_receivable_account_id" ref="fr_pcg_recv_pos"/>
+      <field name="rounding_difference_loss_account_id" ref="pcg_658"/>
```

Comment:
field name should be prefixed with l10n_fr, because it's the guideline for fields used only in localizations